src/bin/edit/localization.rs

```diff
@@ -752,6 +797,7 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
         /* ko      */ "전체 단어",
         /* pt_br   */ "Palavra",
         /* ru      */ "Слово",
+        /* tr      */ "Tam Kelime",
```

Comment:
“Tam sözcük”, kelime sounds archaic to me. macOS also uses “sözcük”, and we also use “sözcük” in KDE translations.